src/js/fluxcontroller.js

```diff
@@ -489,6 +493,10 @@ define(function (require, exports, module) {
                                 transferQueue.active(), transferQueue.pending());
                         }
 
+                        if (action.action.noResetOnFailure) {
+                            nextAction.action.noResetOnFailure = action.action.noResetOnFailure;
```

Comment:
Ew. I do not at all like modifying the action object at runtime. I think it would be better to set a flag in the `FluxController` instance the same way we do for `lockUI`, etc.